tests/simulation/translator/utils/xv15_bet_disk_helper.py

```diff
@@ -346,7 +346,7 @@ def _createBETPolars():
     polars = []
     with imperial_unit_system:
         for radial_index, radial_loc in enumerate(polar_radial_locations):
-            sectional_radiuses.append(radial_loc * u.inch)
```

Comment:
why inch removed?

Reply:
Only creates a list of sectional_radiuses values here and add inch later when defining the BETDisk since all the units are assigned there.